src/AddTodoForm.js

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
 
-export default function AddTodoForm(props) {
+export default function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = React.useState('');
+
+  function handleTitleChange(event) {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  }
+
   function handleAddTodo(event) {
     event.preventDefault();
-    const todoTitle = event.target.title.value;
-    console.log(todoTitle);
-    props.onAddTodo(todoTitle);
-    event.target.reset();
+    onAddTodo({ title: todoTitle, id: Date.now() });
```

Comment:
I know they told you to do the `Date.now()` and then immediately afterwards told you not to so I thought I'd expand on it a bit. Although `Date.now()` does give a time down to the millisecond, there is the chance that in a production app two todos could be created at the exact same time so it's not a reliable unique identifier.